.travis.yml

```diff
@@ -97,29 +128,42 @@ matrix:
 
     allow_failures:
       - env: PYTHON_VERSION=2.7 NUMPY_VERSION=dev SETUP_CMD='test -V'
+             CONDA_DEPENDENCIES='pytz'
+             PIP_DEPENDENCIES=''
       # TODO: these fail for now ... make them work!
       - env: PYTHON_VERSION=2.7 MAIN_CMD='pep8 astroplan --count' SETUP_CMD=''
       - env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.8 SETUP_CMD='test -V'
+             CONDA_DEPENDENCIES='pytz'
+             PIP_DEPENDENCIES=''
       - env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.7 SETUP_CMD='test -V'
+             CONDA_DEPENDENCIES='pytz'
+             PIP_DEPENDENCIES=''
       - env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.6 SETUP_CMD='test -V'
-
-
-install:
-    - source .continuous-integration/travis/setup_environment_$TRAVIS_OS_NAME.sh
-
-script:
-    - if [[ $MAIN_CMD == 'python setup.py' ]]; then
-        $MAIN_CMD $SETUP_CMD;
+             CONDA_DEPENDENCIES='pytz'
+             PIP_DEPENDENCIES=''
+
+before_install:
+    - git clone git://github.com/astropy/ci-helpers.git
+    - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
+
+    # This is needed to make matplotlib plot testing work
```

Comment:
Many packages need this. Move this to `ci-helpers` and make it opt-in, i.e. execute these commands if the user sets something like `SETUP_DISPLAY_FOR_MATPLOTLIB`?